src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -51,15 +43,51 @@ Item {
 
     bgcolor: "transparent"
 
-    onClicked: __pictureSource = platformUtilities.getPicture(qgisProject.homePath + '/DCIM')
+    onClicked: {
+      camloader.active = true
+    }
 
     iconSource: Style.getThemeIcon("ic_camera_alt_border_24dp")
   }
 
-  Connections {
-    target: __pictureSource
-    onPictureReceived: {
-      valueChanged('DCIM/' + path, false)
+  Loader {
+    id: camloader
+    sourceComponent: camcomponent
+    active: false
+  }
+
+  Component {
+    id: camcomponent
+
+    Popup {
+      id: campopup
+
+      Component.onCompleted: open()
+
+      parent: ApplicationWindow.overlay
+
+      x: 0
+      y: 0
+      height: parent.height
+      width: parent.width
+
+      modal: true
+      focus: true
+
+      QField.QFieldCamera {
```

Comment:
I think we should just call that `Camera` to make it a bit shorter

Reply:
I called it QFieldCamera that we know it's a specific QField implementation and there is no confusion with Camera of Multimedia. But actually I don't mind to rename it, because we have already some specific implementations without a prefix (ex. Button)...